dll/win32/syssetup/wizard.c

```diff
@@ -604,6 +742,11 @@ ProductPageDlgProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     {
                         OnChooseOption(hwndDlg, pSetupData->ProductOption);
                         DoWriteProductOption(pSetupData->ProductOption);
+                        if (PRODUCT_OPTION_DEFAULT == PRODUCT_OPTION_WORKSTATION)
```

Comment:
You are comparing two constants (yes, I see same code pattern in this file, no idea how this heresy got into master)

Reply:
Removed.